### DISCUSSION TITLE
How to set Color mode using a script?

### DISCUSSION THREAD

**Discussion by vivaanmathur** [chakra-ui/chakra-ui#6032] [Q&A]
My JavaScript code uses an axios request that checks whether dark mode or light mode is to be used (the server returns true or false).
```
  if(result == "dark"){
// set colour mode here
  }
  else{
// set light mode here
  }
```

How to set Color mode in Chakra UI using JavaScript?
There is very little documentation I could find online, with most of it referring to the useColorMode React hook which can't be used as a function. I am using Create React App (if that info is needed)

**Comment by aroyan** [comment]
Are you going to set dark mode or light mode based on data on server?

**Reply by vivaanmathur** [reply]
Yes, it fetches the user settings from a nodejs server.
